IFC4x3/PropertyConstants/w/WELDEDANDINSERTABLETOCONTINOUSWELDEDRAIL_3eSc38OQzCyB8vnIaNQTl9.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="WELDEDANDINSERTABLETOCONTINOUSWELDEDRAIL" UniqueId="e87260c8-61af-4cf0-b239-c5291769dbc9" />
```

Comment:
Difficult to read.

Reply:
Done : 426664c10f4b7a9f9b1eff4dddaa4ee62a65d0e3